File: app/src/main/java/com/coderstory/flyme/utils/Application.java
package com.coderstory.flyme.utils;


public class Application extends android.app.Application {
    private static Application mInstance;

    public static Application getInstance() {
        return mInstance;
    }

    @Override
    public void onCreate() {
        super.onCreate();
        mInstance = this;
        System.loadLibrary("Utils");
        Cpp.initCpp(mInstance);
    }
}

File: app/src/main/java/com/coderstory/flyme/tools/Application.kt
package com.coderstory.flyme.tools

import android.app.Application
import android.content.Context

class Application : Application() {
    override fun onCreate() {
        super.onCreate()
        mInstance = this
        System.loadLibrary("Utils")

    }

    companion object {
        private lateinit var mInstance: Context
        val instance: Context
            get() = mInstance
    }
}